thread_options.h

```diff
@@ -620,6 +621,9 @@ struct thread_options_pack {
 	fio_fp64_t latency_percentile;
 	uint32_t latency_run;
 
+
+	uint32_t prepopulate;
```

Comment:
I wanted to put this just next to fallocate option (above in the file), but I've got static assert in libfio.c:367.
And I'm not even sure if we need this - pls verify if I can do better here...

Reply:
Theoretically you can do it if you but you'll need to add some the padding because there are some variables that HAVE to be aligned on 64 bit boundaries and that's what the static assert is grumbling about. It's not great to have a lot of holes but make sure you rebase your patch as a change to this data structure happened recently and maybe that makes it easier to put this where you want (but it may not).

Reply:
I believe I don't have to worry about it anymore.